WiFiManager.cpp

```diff
@@ -278,40 +335,37 @@ uint8_t WiFiManager::waitForConnectResult() {
 }
 
 void WiFiManager::startWPS() {
-  DEBUG_WM("START WPS");
+  DEBUG_WM(F("START WPS"));
   WiFi.beginWPSConfig();
-  DEBUG_WM("END WPS");
-}
-/*
-  String WiFiManager::getSSID() {
-  if (_ssid == "") {
-    DEBUG_WM(F("Reading SSID"));
-    _ssid = WiFi.SSID();
-    DEBUG_WM(F("SSID: "));
-    DEBUG_WM(_ssid);
-  }
-  return _ssid;
-  }
+  DEBUG_WM(F("END WPS"));
+}
+//Convenient for debugging but wasteful of program space.
+//Remove if short of space
+char* WiFiManager::getStatus(int status)
+{
+   switch (status)
```

Comment:
All of the values returned by ESP8266WIFISTA are correctly identified. However there is some extras which are WL_NO_SHIELD, WL_SCAN_COMPLETED, WL_CONNECTION_LOST. I'll remove them in the next version and if any extra values are added to ESP8266WiFiSTAClass::status() they will become the default value which is UNKNOWN